apps/web/components/Regulations/RegulationChangelog.tsx

```diff
@@ -115,19 +115,24 @@ export const useRegulationEffectPrepper = (
             { name },
           )
           const href =
-            item.effect === 'amend'
+            item.effect === 'amend' && item.status === 'published'
               ? linkToRegulation(regulation.name, {
                   d: item.date,
                   diff: true,
                 })
+              : item.effect === 'amend' && item.status === 'pending'
+              ? linkToRegulation(item.name)
```

Comment:
```suggestion
              // Link to current version (no diff) because the impact (text-changes)
              // has not been inserted yet.
              ? linkToRegulation(item.name)
```